extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java

```diff
@@ -121,16 +119,16 @@ public Set<AttributeDesignatorType> getProvidedAttributes() {
      */
     @Override
     public <AV extends AttributeValue> AttributeBag<AV> get(AttributeFqn attributeGUID, Datatype<AV> attributeDatatype,
-                                                            EvaluationContext context, Optional<EvaluationContext> optionalEvaluationContext) throws IndeterminateEvaluationException {
-        String id = attributeGUID.getId();
-        String subject = findSubjectInEnvironmentContext(context);
+        EvaluationContext context, Optional<EvaluationContext> optionalEvaluationContext) throws
+            IndeterminateEvaluationException {String id = attributeGUID.getId();String subject =
```

Comment:
Looks like this command needs a line break(?)

Reply:
Corrected this!